site/content/docs/5.2/components/toasts.md

```diff
@@ -184,7 +184,7 @@ Alternatively, you can also add additional controls and components to toasts.
 
 Building on the above example, you can create different toast color schemes with our [color]({{< docsref "/utilities/colors" >}}) and [background]({{< docsref "/utilities/background" >}}) utilities. Here we've added `.text-bg-secondary` to the `.toast`, and then added `.btn-close-white` to our close button. For a crisp edge, we remove the default border with `.border-0`.
 
-{{< example class="bg-light" >}}
+{{< example >}}
 <div class="toast align-items-center text-bg-secondary border-0" role="alert" aria-live="assertive" aria-atomic="true">
   <div class="d-flex">
     <div class="toast-body my-auto">
```

Comment:
Missing some changes below to change `bg-dark` to `bg-body-secondary`. Is that a miss or is it wanted ?

Reply:
Totally a miss. Good catch! Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/16d0b84126426aa71b6dc6d2521e675e3875ae52.
Just not included the `.rounded-whatever`.